Makefile

```diff
@@ -8,7 +8,6 @@ lint:
 	golangci-lint run ./...
 
 build:
-	$(MAKE) -C scripts
```

Comment:
@snprajwal Would you be able to add a README.md file in `scripts/` or `magic/` that explains what is `magic/magic.go` and how to generate a new version?